### DISCUSSION TITLE
cypress teamcity integration parallel

### DISCUSSION THREAD

**Discussion by dev6testing** [cypress-io/cypress#21244] [Questions and Help]
dose anyone know how to integrate cypress with teamcity so tests can run in parallel?

**Comment by matthiasbaldi** [comment]
I think only over the cloud based Cypress API
Like: `cypress run --parallel --record --key <anystring> --ci-build-id <uniqueBuildId>`

Maybe that is something for Cypress to integrate: https://www.jetbrains.com/help/teamcity/2022.04/parallel-tests.html
wdyt @jennifer-shehane?
